war/src/main/webapp/help/system-config/master-slave/usage.html

```diff
@@ -6,7 +6,8 @@
       Utilize this node as much as possible
```

Comment:
Yeah, I updated the table after submitting the PR :)

Also, I hadn't changed this yet, as it has a dependency on a `.properties` translation.

I also wanted to ask in general, if we can make this documentation dynamic, since this help text depends on what's in the launchers drop-down.